<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarCardViewModel</class>
 <widget class="QWidget" name="CarCardViewModel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
background-color: rgb(246, 255, 212);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,0,0,0,0">
   <item>
    <widget class="QLabel" name="carImage">
     <property name="text">
      <string>CarImage</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="carBrandName">
     <property name="styleSheet">
      <string notr="true">QLabel {
	border: 1.5px solid rgb(180,226,155);
	border-radius: 50px;
}QLabel{
	font-size: 10pt;
	font-weight: 600;
}</string>
     </property>
     <property name="text">
      <string>CarBrand</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="carModelName">
     <property name="styleSheet">
      <string notr="true">QLabel {
	border: 1.5px solid rgb(180,226,155);
	border-radius: 50px;
}QLabel{
	font-size: 10pt;
	font-weight: 600;
}</string>
     </property>
     <property name="text">
      <string>CarModel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="styleSheet">
        <string notr="true">QLabel {
	border: 1.5px solid rgb(180,226,155);
	border-radius: 50px;
}QLabel{
	font-size: 10pt;
	font-weight: 600;
}</string>
       </property>
       <property name="text">
        <string>Класс авто</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="carClass">
       <property name="styleSheet">
        <string notr="true">QLabel {
	border: 1.5px solid rgb(180,226,155);
	border-radius: 50px;
}QLabel{
	font-size: 10pt;
	font-weight: 600;
}</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_10">
       <property name="styleSheet">
        <string notr="true">QLabel {
	border: 1.5px solid rgb(180,226,155);
	border-radius: 50px;
}QLabel{
	font-size: 10pt;
	font-weight: 600;
}</string>
       </property>
       <property name="text">
        <string>Разгон до 100 км/ч</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="carAcseleration">
       <property name="styleSheet">
        <string notr="true">QLabel {
	border: 1.5px solid rgb(180,226,155);
	border-radius: 50px;
}QLabel{
	font-size: 10pt;
	font-weight: 600;
}</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_12">
       <property name="styleSheet">
        <string notr="true">QLabel {
	border: 1.5px solid rgb(180,226,155);
	border-radius: 50px;
}QLabel{
	font-size: 10pt;
	font-weight: 600;
}</string>
       </property>
       <property name="text">
        <string>Мощность, лс</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="carPower">
       <property name="styleSheet">
        <string notr="true">QLabel {
	border: 1.5px solid rgb(180,226,155);
	border-radius: 50px;
}QLabel{
	font-size: 10pt;
	font-weight: 600;
}</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_14">
       <property name="styleSheet">
        <string notr="true">QLabel {
	border: 1.5px solid rgb(180,226,155);
	border-radius: 50px;
}QLabel{
	font-size: 10pt;
	font-weight: 600;
}</string>
       </property>
       <property name="text">
        <string>Стоимость, руб/сутки</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="rentalPrice">
       <property name="styleSheet">
        <string notr="true">QLabel {
	border: 1.5px solid rgb(180,226,155);
	border-radius: 50px;
}QLabel{
	font-size: 10pt;
	font-weight: 600;
}</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="3">
     <item>
      <widget class="QPushButton" name="createOrder">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: #4CAF50;
	color: white;
	border: none;
	text-align: center;
	text-decoration: none;
	display: inline-block;
	font-size: 20px;
	cursor: pointer;
	border-radius: 12px;
}

QPushButton:hover{
	background-color: #45a049;
}

QPushButton:pressed{
	background-color: #3e8e41;
}</string>
       </property>
       <property name="text">
        <string>Сделать заявку</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
